Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1467/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #526 of commit abfbed8c1af746724fff74f7c5c88ef730e0bf53 automatically merged.
GitHub pull request #526 of commit abfbed8c1af746724fff74f7c5c88ef730e0bf53 automatically merged.
GitHub pull request #526 of commit abfbed8c1af746724fff74f7c5c88ef730e0bf53 automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace@3
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace@3/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git4928045154649829763.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git8530080865090665238.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/526/merge^{commit} # timeout=10
 > git branch -a --contains c211ce81b1a22709843ef9bcabcd240e937d1424 # timeout=10
 > git rev-parse remotes/origin/pr/526/merge^{commit} # timeout=10
Checking out Revision c211ce81b1a22709843ef9bcabcd240e937d1424 (origin/pr/526/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f c211ce81b1a22709843ef9bcabcd240e937d1424
First time build. Skipping changelog.
[workspace@3] $ /bin/sh -xe /tmp/hudson1382406288058479528.sh
+ rm -Rf /var/lib/jenkins/jobs/PR_BUILDER/workspace@3/vm
+ cp -R ../vm /var/lib/jenkins/jobs/PR_BUILDER/workspace@3/vm
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace@3/vm
+ git pull
Already up-to-date.
[workspace@3] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace@3/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory6842501470140455416.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace@3 -e build_number=1467 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace@3/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace@3/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace@3] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace@3/vm/cibox/jobs/build.yml -i /tmp/inventory6455385326146431410.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace@3 -e build_number=1467 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace@3] $ ansible-playbook /var/www/build1467/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1467 -e drupal_folder=/var/www/build1467 -e site_url=http://openy-dev.ffwua.com/build1467 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
skipping: [localhost]

TASK: [Remove settings.php] *************************************************** 
skipping: [localhost]

TASK: [Creating settings.php] ************************************************* 
skipping: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
skipping: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/default/files'})
skipping: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
skipping: [localhost]

TASK: [Creating drupal database] ********************************************** 
skipping: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
skipping: [localhost]

TASK: [Importing from local backup] ******************************************* 
skipping: [localhost]

TASK: [Remove local backup from build folder] ********************************* 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
skipping: [localhost]

TASK: [Clear Drupal cache] **************************************************** 
skipping: [localhost]

TASK: [Updating database] ***************************************************** 
skipping: [localhost]

TASK: [Apply pending entity schema updates] *********************************** 
skipping: [localhost]

TASK: [Download stage file proxy] ********************************************* 
skipping: [localhost]

TASK: [Enable stage file proxy] *********************************************** 
skipping: [localhost]

TASK: [Set configuration for stage file proxy] ******************************** 
skipping: [localhost]

TASK: [Set folder for stage file proxy] *************************************** 
skipping: [localhost]

TASK: [Updates listing] ******************************************************* 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
changed: [localhost]

TASK: [Installing drupal] ***************************************************** 
changed: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
ok: [localhost]

TASK: [Set writable private dir] ********************************************** 
ok: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
ok: [localhost]

TASK: [Copy settings to tmp folder (WINDOWS)] ********************************* 
skipping: [localhost]

TASK: [Add general post-settings to settings.php] ***************************** 
changed: [localhost] => (item={'status': True, 'name': '$settings["pp_environment"]', 'value': u'default'})

TASK: [Add environment specific post-settings] ******************************** 
changed: [localhost] => (item={'status': True, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': False, 'name': '$settings["cache"]["bins"]["render"]', 'value': 'cache.backend.memory'})
skipping: [localhost] => (item={'status': False, 'name': '$config["stage_file_proxy.settings"]["origin"]', 'value': u'http://openy-dev.ffwua.com/demo_upgrade_path'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Enable general modules] ************************************************ 
skipping: [localhost]

TASK: [Enable environment specific modules] *********************************** 
skipping: [localhost] => (item={'status': False, 'name': 'devel shield'})

TASK: [Run general drush commands] ******************************************** 
skipping: [localhost]

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'pm-uninstall', 'arguments': 'update -y'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=openy'})
changed: [localhost] => (item={'status': True, 'name': 'cr', 'arguments': ''})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'shield.settings user admin -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'shield.settings pass openy -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': 'system.site uuid USER_YOUR_UUID_HERE -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'stage_file_proxy.settings origin http://openy-dev.ffwua.com/demo_upgrade_path -y'})
changed: [localhost] => (item={'status': True, 'name': 'config-set', 'arguments': "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"})

TASK: [Debug] ***************************************************************** 
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y pm-uninstall update -y', 'end': '2017-06-11 13:21:54.611152', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nupdate was successfully uninstalled.                                 \x1b[1;32;40m\x1b[1m[ok]\x1b[0m', 'stdout': 'The following extensions will be uninstalled: update\nDo you really want to continue? (y/n): y', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'pm-uninstall', 'arguments': 'update -y'}, 'warnings': [], 'delta': '0:00:12.409523', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y pm-uninstall update -y'}, 'stdout_lines': ['The following extensions will be uninstalled: update', 'Do you really want to continue? (y/n): y'], 'start': '2017-06-11 13:21:42.201629'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y pm-uninstall update -y",
        "delta": "0:00:12.409523",
        "end": "2017-06-11 13:21:54.611152",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y pm-uninstall update -y",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "update -y",
            "name": "pm-uninstall",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:21:42.201629",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nupdate was successfully uninstalled.                                 \u001b[1;32;40m\u001b[1m[ok]\u001b[0m",
        "stdout": "The following extensions will be uninstalled: update\nDo you really want to continue? (y/n): y",
        "stdout_lines": [
            "The following extensions will be uninstalled: update",
            "Do you really want to continue? (y/n): y"
        ],
        "warnings": []
    },
    "var": {
        "item.stdout": "The following extensions will be uninstalled: update\nDo you really want to continue? (y/n): y"
    }
}
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y user-password admin --password=openy', 'end': '2017-06-11 13:21:57.984820', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nChanged password for admin                                           \x1b[1;32;40m\x1b[1m[success]\x1b[0m', 'stdout': u'', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'user-password', 'arguments': u'admin --password=openy'}, 'warnings': [], 'delta': '0:00:03.197662', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y user-password admin --password=openy'}, 'stdout_lines': [], 'start': '2017-06-11 13:21:54.787158'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y user-password admin --password=openy",
        "delta": "0:00:03.197662",
        "end": "2017-06-11 13:21:57.984820",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y user-password admin --password=openy",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "admin --password=openy",
            "name": "user-password",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:21:54.787158",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nChanged password for admin                                           \u001b[1;32;40m\u001b[1m[success]\u001b[0m",
        "stdout": "",
        "stdout_lines": [],
        "warnings": []
    },
    "var": {
        "item.stdout": ""
    }
}
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y cr', 'end': '2017-06-11 13:22:19.438047', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nCache rebuild complete.                                              \x1b[1;32;40m\x1b[1m[ok]\x1b[0m\nDirectory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.', 'stdout': u'', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'cr', 'arguments': ''}, 'warnings': [], 'delta': '0:00:21.211003', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y cr'}, 'stdout_lines': [], 'start': '2017-06-11 13:21:58.227044'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y cr",
        "delta": "0:00:21.211003",
        "end": "2017-06-11 13:22:19.438047",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y cr",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "",
            "name": "cr",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:21:58.227044",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nCache rebuild complete.                                              \u001b[1;32;40m\u001b[1m[ok]\u001b[0m\nDirectory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.",
        "stdout": "",
        "stdout_lines": [],
        "warnings": []
    },
    "var": {
        "item.stdout": ""
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={'cmd': "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y", 'end': '2017-06-11 13:22:21.835187', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.', 'stdout': 'Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'config-set', 'arguments': "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"}, 'warnings': [], 'delta': '0:00:02.184085', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u"APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"}, 'stdout_lines': ['Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y'], 'start': '2017-06-11 13:22:19.651102'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
        "delta": "0:00:02.184085",
        "end": "2017-06-11 13:22:21.835187",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
            "name": "config-set",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:22:19.651102",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.",
        "stdout": "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y",
        "stdout_lines": [
            "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y"
        ],
        "warnings": []
    },
    "var": {
        "item.stdout": "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y"
    }
}

PLAY RECAP ******************************************************************** 
localhost                  : ok=28   changed=12   unreachable=0    failed=0   

[workspace@3] $ ansible-playbook /var/www/build1467/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal_upgrade1467 -e drupal_folder=/var/www/build1467 -e site_url=http://upgrade.openy-dev.ffwua.com/build1467 -e pp_environment=default -e run_reinstall=true -e workflow_type=sql -e settings_site_folder=upgrade -e trigger_drush_make=false -e stage_file_proxy=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
ok: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Creating settings.php] ************************************************* 
changed: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
changed: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
changed: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
changed: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
changed: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
changed: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/upgrade/files'})
changed: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
changed: [localhost]

TASK: [Importing from local backup] ******************************************* 
changed: [localhost]

TASK: [Remove local backup from build folder] ********************************* 
changed: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
skipping: [localhost]

TASK: [Clear Drupal cache] **************************************************** 
changed: [localhost]

TASK: [Updating database] ***************************************************** 
changed: [localhost]

TASK: [Apply pending entity schema updates] *********************************** 
changed: [localhost]

TASK: [Download stage file proxy] ********************************************* 
changed: [localhost]

TASK: [Enable stage file proxy] *********************************************** 
changed: [localhost]

TASK: [Set configuration for stage file proxy] ******************************** 
changed: [localhost]

TASK: [Set folder for stage file proxy] *************************************** 
changed: [localhost]

TASK: [Updates listing] ******************************************************* 
ok: [localhost] => {
    "var": {
        "updates.stdout_lines": [
            "The following updates are pending:",
            "",
            "paragraphs module : ",
            "  8009 -   Set the weight to 11 to override content_translation's hook_module_implements_alter implementation ",
            "  8010 -   Add behavior plugins fields. ",
            "  8011 -   Make the behavior plugins field of Paragraphs revisionable. ",
            "",
            "Do you wish to run all pending updates? (y/n): y"
        ]
    }
}

TASK: [Droping drupal database] *********************************************** 
skipping: [localhost]

TASK: [Creating drupal database] ********************************************** 
skipping: [localhost]

TASK: [Remove settings.php] *************************************************** 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Installing drupal] ***************************************************** 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
ok: [localhost]

TASK: [Set writable private dir] ********************************************** 
ok: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
ok: [localhost]

TASK: [Copy settings to tmp folder (WINDOWS)] ********************************* 
skipping: [localhost]

TASK: [Add general post-settings to settings.php] ***************************** 
changed: [localhost] => (item={'status': True, 'name': '$settings["pp_environment"]', 'value': u'default'})

TASK: [Add environment specific post-settings] ******************************** 
changed: [localhost] => (item={'status': True, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': False, 'name': '$settings["cache"]["bins"]["render"]', 'value': 'cache.backend.memory'})
skipping: [localhost] => (item={'status': False, 'name': '$config["stage_file_proxy.settings"]["origin"]', 'value': u'http://openy-dev.ffwua.com/demo_upgrade_path'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Enable general modules] ************************************************ 
skipping: [localhost]

TASK: [Enable environment specific modules] *********************************** 
skipping: [localhost] => (item={'status': False, 'name': 'devel shield'})

TASK: [Run general drush commands] ******************************************** 
skipping: [localhost]

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'pm-uninstall', 'arguments': 'update -y'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=openy'})
changed: [localhost] => (item={'status': True, 'name': 'cr', 'arguments': ''})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'shield.settings user admin -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'shield.settings pass openy -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': 'system.site uuid USER_YOUR_UUID_HERE -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'stage_file_proxy.settings origin http://openy-dev.ffwua.com/demo_upgrade_path -y'})
changed: [localhost] => (item={'status': True, 'name': 'config-set', 'arguments': "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"})

TASK: [Debug] ***************************************************************** 
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y pm-uninstall update -y', 'end': '2017-06-11 13:24:31.684855', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nupdate is already uninstalled.                                       \x1b[1;32;40m\x1b[1m[ok]\x1b[0m\nThere were no extensions that could be uninstalled.                  \x1b[1;32;40m\x1b[1m[ok]\x1b[0m', 'stdout': u'', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'pm-uninstall', 'arguments': 'update -y'}, 'warnings': [], 'delta': '0:00:02.319527', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y pm-uninstall update -y'}, 'stdout_lines': [], 'start': '2017-06-11 13:24:29.365328'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y pm-uninstall update -y",
        "delta": "0:00:02.319527",
        "end": "2017-06-11 13:24:31.684855",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y pm-uninstall update -y",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "update -y",
            "name": "pm-uninstall",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:24:29.365328",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nupdate is already uninstalled.                                       \u001b[1;32;40m\u001b[1m[ok]\u001b[0m\nThere were no extensions that could be uninstalled.                  \u001b[1;32;40m\u001b[1m[ok]\u001b[0m",
        "stdout": "",
        "stdout_lines": [],
        "warnings": []
    },
    "var": {
        "item.stdout": ""
    }
}
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y user-password admin --password=openy', 'end': '2017-06-11 13:24:34.699851', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nChanged password for admin                                           \x1b[1;32;40m\x1b[1m[success]\x1b[0m', 'stdout': u'', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'user-password', 'arguments': u'admin --password=openy'}, 'warnings': [], 'delta': '0:00:02.803673', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y user-password admin --password=openy'}, 'stdout_lines': [], 'start': '2017-06-11 13:24:31.896178'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y user-password admin --password=openy",
        "delta": "0:00:02.803673",
        "end": "2017-06-11 13:24:34.699851",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y user-password admin --password=openy",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "admin --password=openy",
            "name": "user-password",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:24:31.896178",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nChanged password for admin                                           \u001b[1;32;40m\u001b[1m[success]\u001b[0m",
        "stdout": "",
        "stdout_lines": [],
        "warnings": []
    },
    "var": {
        "item.stdout": ""
    }
}
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y cr', 'end': '2017-06-11 13:24:50.932285', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nCache rebuild complete.                                              \x1b[1;32;40m\x1b[1m[ok]\x1b[0m\nDirectory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.', 'stdout': u'', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'cr', 'arguments': ''}, 'warnings': [], 'delta': '0:00:16.084994', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y cr'}, 'stdout_lines': [], 'start': '2017-06-11 13:24:34.847291'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y cr",
        "delta": "0:00:16.084994",
        "end": "2017-06-11 13:24:50.932285",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y cr",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "",
            "name": "cr",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:24:34.847291",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nCache rebuild complete.                                              \u001b[1;32;40m\u001b[1m[ok]\u001b[0m\nDirectory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.",
        "stdout": "",
        "stdout_lines": [],
        "warnings": []
    },
    "var": {
        "item.stdout": ""
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={'cmd': "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y", 'end': '2017-06-11 13:24:54.018554', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.', 'stdout': 'Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'config-set', 'arguments': "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"}, 'warnings': [], 'delta': '0:00:02.832894', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u"APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"}, 'stdout_lines': ['Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y'], 'start': '2017-06-11 13:24:51.185660'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
        "delta": "0:00:02.832894",
        "end": "2017-06-11 13:24:54.018554",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
            "name": "config-set",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-11 13:24:51.185660",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.",
        "stdout": "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y",
        "stdout_lines": [
            "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y"
        ],
        "warnings": []
    },
    "var": {
        "item.stdout": "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y"
    }
}

PLAY RECAP ******************************************************************** 
localhost                  : ok=40   changed=23   unreachable=0    failed=0   

[workspace@3] $ ansible-playbook /var/www/build1467/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_user=root -e mysql_db=drupal_install1467 -e drupal_folder=/var/www/build1467 -e site_url=http://install.openy-dev.ffwua.com/build1467 -e pp_environment=default -e run_reinstall=true -e settings_site_folder=install -e trigger_drush_make=false -e run_installation_process=false -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
ok: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Creating settings.php] ************************************************* 
changed: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
changed: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
changed: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
changed: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
changed: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
changed: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/install/files'})
changed: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
skipping: [localhost]

TASK: [Creating drupal database] ********************************************** 
skipping: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
skipping: [localhost]

TASK: [Importing from local backup] ******************************************* 
skipping: [localhost]

TASK: [Remove local backup from build folder] ********************************* 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
skipping: [localhost]

TASK: [Clear Drupal cache] **************************************************** 
skipping: [localhost]

TASK: [Updating database] ***************************************************** 
skipping: [localhost]

TASK: [Apply pending entity schema updates] *********************************** 
skipping: [localhost]

TASK: [Download stage file proxy] ********************************************* 
skipping: [localhost]

TASK: [Enable stage file proxy] *********************************************** 
skipping: [localhost]

TASK: [Set configuration for stage file proxy] ******************************** 
skipping: [localhost]

TASK: [Set folder for stage file proxy] *************************************** 
skipping: [localhost]

TASK: [Updates listing] ******************************************************* 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Remove settings.php] *************************************************** 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Installing drupal] ***************************************************** 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
failed: [localhost] => {"failed": true, "parsed": false}
BECOME-SUCCESS-wskmjwltzvemadvlsxphzbawgzpztboj
Traceback (most recent call last):
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1497187503.79-113030655785600/file", line 2012, in <module>
    main()
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1497187503.79-113030655785600/file", line 291, in main
    changed |= recursive_set_attributes(module, file_args['path'], follow, file_args)
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1497187503.79-113030655785600/file", line 132, in recursive_set_attributes
    changed |= module.set_fs_attributes_if_different(tmp_file_args, changed)
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1497187503.79-113030655785600/file", line 1207, in set_fs_attributes_if_different
    file_args['path'], file_args['mode'], changed
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1497187503.79-113030655785600/file", line 1045, in set_mode_if_different
    path_stat = os.lstat(path)
OSError: [Errno 2] No such file or directory: '/tmp/make_tmp_1497187459_593d44837e5a9/__build__/modules/contrib/paragraphs/modules/paragraphs_demo/config/install/core.entity_view_display.paragraph.user.default.yml'


FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=17   changed=8    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

--- a/modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.paragraph.session_time.field_session_time_days.yml
+++ b/modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.paragraph.session_time.field_session_time_days.yml
@@ -11,15 +11,7 @@ field_name: field_session_time_days
 entity_type: paragraph
 bundle: session_time
 label: Days
-description: 'Checkboxes with following values:
-                  - sunday|Sunday
-                  - monday|Monday
-                  - tuesday|Tuesday
-                  - wednesday|Wednesday
-                  - thursday|Thursday
-                  - friday|Friday
-                  - saturday|Saturday
-              Should support multiple values.'
+description: 'Checkboxes with following values: <ul><li>sunday|Sunday</li><li>monday|Monday</li><li>tuesday|Tuesday</li><li>wednesday|Wednesday</li><li>thursday|Thursday</li><li>friday|Friday</li><li>saturday|Saturday</li></ul><br/ >Should support multiple values.'
 required: false
 translatable: true
 default_value: {  }

--- a/modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.paragraph.session_time.field_session_time_days.yml
+++ b/modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.paragraph.session_time.field_session_time_days.yml
@@ -11,7 +11,7 @@ field_name: field_session_time_days
 entity_type: paragraph
 bundle: session_time
 label: Days
-description: 'Checkboxes with following values: <ul><li>sunday|Sunday</li><li>monday|Monday</li><li>tuesday|Tuesday</li><li>wednesday|Wednesday</li><li>thursday|Thursday</li><li>friday|Friday</li><li>saturday|Saturday</li></ul><br/ >Should support multiple values.'
+description: 'Checkboxes with following values: sunday|Sunday monday|Monday tuesday|Tuesday wednesday|Wednesday thursday|Thursday friday|Friday saturday|Saturday. Should support multiple values.'
 required: false
 translatable: true
 default_value: {  }

